CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 **BREAKING CHANGES**
 
+# matrix-sdk-crypto-wasm v13.0.0
+
+**BREAKING CHANGES**
+
 -   Update matrix-rusk-sdk to `0.9.0`.
 -   Expose new API `DehydratedDevices.getDehydratedDeviceKey`, `DehydratedDevices.saveDehydratedDeviceKey`
```

Comment:
while you're working on this file, could you fix the points I raised at https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/pull/179/files#r1904474818 ?